packages/cxl-jwplayer/src/components/cxl-jwplayer.js

```diff
@@ -1,17 +1,152 @@
-import { LitElement, html, customElement } from 'lit-element';
+import { LitElement, html, customElement, property } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+// import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
 import cxlJwplayerStyles from '../styles/cxl-jwplayer-css.js';
+// import cxlJwplayerGlobalStyles from '../styles/global/cxl-jwplayer-css.js';
 
 @customElement('cxl-jwplayer')
 export class CxlJwPlayer extends LitElement {
   static get styles() {
     return [cxlJwplayerStyles];
   }
 
-  static get properties() {
-    return {};
+  @property({ type: Object })
+  player = null;
+
+  @property({ type: Object })
+  config = {};
+
+  @property({ type: String })
+  mediaId = '';
+
+  constructor() {
+    super();
+    this.jwplayer = window.jwplayer || null;
+    if (!this.jwplayer) console.error('JWPlayer library not found.');
+  }
+
+  /*
+   * Implemented to avoid default, which attaches a shadowDOM tree byt default.
+   */
+  createRenderRoot() {
+    return this;
   }
 
   render() {
-    return html``;
+    return html`
+      <h2>cxl-jwplayer</h2>
+      <div id="cxl-jwplayer-container" class="cxl-jwplayer-container">
```

Comment:
The media ids are missing from ids and casing is different. This will cause significant refactor of all plugins. 

Not a problem, just something to think about when things don't work.

Reply:
I'm putting them back in to avoid breaking things. We could trust custom elements scope, but if the PHP plugin depends on it, there's no point in taking it out.

Reply:
Cool. I'm thinking towards how this will work off we separate out the transcript viewer as its own component and how the jw plugin will ref the component.